SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -3,11 +3,10 @@
 #include "CoreMinimal.h"
 
 #include "Utils/SchemaOption.h"
+#include "SpatialCommonTypes.h"
 
 #include <cstdint>
```

Comment:
```suggestion
```

Reply:
I am going to remove SpatialCommonTypes and cstdint and include c_worker.h.